### ISSUE TITLE
2024-12-04, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#10]
# Today's Arxiv Papers

(1) [T-REG: Preference Optimization with Token-Level Reward Regularization](http://arxiv.org/abs/2412.02685v1), 2024-12-03

(2) [Mind the Gap: Examining the Self-Improvement Capabilities of Large Language Models](http://arxiv.org/abs/2412.02674v1), 2024-12-03

(3) [LLM-Enhanced Path Planning: Safe and Efficient Autonomous Navigation with Instructional Inputs](http://arxiv.org/abs/2412.02655v1), 2024-12-03

(4) [QA-TOOLBOX: Conversational Question-Answering for process task guidance in manufacturing](http://arxiv.org/abs/2412.02638v1), 2024-12-03

(5) [Time-Reversal Provides Unsupervised Feedback to LLMs](http://arxiv.org/abs/2412.02626v1), 2024-12-03

(6) [Improving Dynamic Object Interactions in Text-to-Video Generation with AI Feedback](http://arxiv.org/abs/2412.02617v1), 2024-12-03

(7) [AV-Odyssey Bench: Can Your Multimodal LLMs Really Understand Audio-Visual Information?](http://arxiv.org/abs/2412.02611v1), 2024-12-03

(8) [Interpretable Company Similarity with Sparse Autoencoders](http://arxiv.org/abs/2412.02605v1), 2024-12-03

(9) [CEGI: Measuring the trade-off between efficiency and carbon emissions for SLMs and VLMs](http://arxiv.org/abs/2412.02602v1), 2024-12-03

(10) [PrefixLLM: LLM-aided Prefix Circuit Design](http://arxiv.org/abs/2412.02594v1), 2024-12-03

(11) [OCR Hinders RAG: Evaluating the Cascading Impact of OCR on Retrieval-Augmented Generation](http://arxiv.org/abs/2412.02592v1), 2024-12-03

(12) [Explainable CTR Prediction via LLM Reasoning](http://arxiv.org/abs/2412.02588v1), 2024-12-03

(13) [Semantic Tokens in Retrieval Augmented Generation](http://arxiv.org/abs/2412.02563v1), 2024-12-03

(14) [Patent-CR: A Dataset for Patent Claim Revision](http://arxiv.org/abs/2412.02549v1), 2024-12-03

(15) [LLMForecaster: Improving Seasonal Event Forecasts with Unstructured Textual Data](http://arxiv.org/abs/2412.02525v1), 2024-12-03

(16) [DP-2Stage: Adapting Language Models as Differentially Private Tabular Data Generators](http://arxiv.org/abs/2412.02467v1), 2024-12-03

(17) [Gracefully Filtering Backdoor Samples for Generative Large Language Models without Retraining](http://arxiv.org/abs/2412.02454v1), 2024-12-03

(18) [GerPS-Compare: Comparing NER methods for legal norm analysis](http://arxiv.org/abs/2412.02427v1), 2024-12-03

(19) [A Multi-Agent Framework for Extensible Structured Text Generation in PLCs](http://arxiv.org/abs/2412.02410v1), 2024-12-03

(20) [ScImage: How Good Are Multimodal Large Language Models at Scientific Text-to-Image Generation?](http://arxiv.org/abs/2412.02368v1), 2024-12-03

(21) [Large Multimodal Agents for Accurate Phishing Detection with Enhanced Token Optimization and Cost Reduction](http://arxiv.org/abs/2412.02301v1), 2024-12-03

(22) [A Comprehensive Evaluation of Large Language Models on Aspect-Based Sentiment Analysis](http://arxiv.org/abs/2412.02279v1), 2024-12-03

(23) [MediaSpin: Exploring Media Bias Through Fine-Grained Analysis of News Headlines](http://arxiv.org/abs/2412.02271v1), 2024-12-03

(24) [Connecting Large Language Models with Blockchain: Advancing the Evolution of Smart Contracts from Automation to Intelligence](http://arxiv.org/abs/2412.02263v1), 2024-12-03

(25) [Compressing KV Cache for Long-Context LLM Inference with Inter-Layer Attention Similarity](http://arxiv.org/abs/2412.02252v1), 2024-12-03

(26) [BANER: Boundary-Aware LLMs for Few-Shot Named Entity Recognition](http://arxiv.org/abs/2412.02228v1), 2024-12-03

(27) [Unlocking Tuning-Free Few-Shot Adaptability in Visual Foundation Models by Recycling Pre-Tuned LoRAs](http://arxiv.org/abs/2412.02220v1), 2024-12-03

(28) [DataLab: A Unifed Platform for LLM-Powered Business Intelligence](http://arxiv.org/abs/2412.02205v1), 2024-12-03

(29) [3D representation in 512-Byte:Variational tokenizer is the key for autoregressive 3D generation](http://arxiv.org/abs/2412.02202v1), 2024-12-03

(30) [LayoutVLM: Differentiable Optimization of 3D Layout via Vision-Language Models](http://arxiv.org/abs/2412.02193v1), 2024-12-03

(31) [Keeping Experts in the Loop: Expert-Guided Optimization for Clinical Data Classification using Large Language Models](http://arxiv.org/abs/2412.02173v1), 2024-12-03

(32) [Jailbreak Defense in a Narrow Domain: Limitations of Existing Methods and a New Transcript-Classifier Approach](http://arxiv.org/abs/2412.02159v1), 2024-12-03

(33) [CausalMob: Causal Human Mobility Prediction with LLMs-derived Human Intentions toward Public Events](http://arxiv.org/abs/2412.02155v1), 2024-12-03

(34) [Leveraging Large Language Models for Comparative Literature Summarization with Reflective Incremental Mechanisms](http://arxiv.org/abs/2412.02149v1), 2024-12-03

(35) [Personalized Multimodal Large Language Models: A Survey](http://arxiv.org/abs/2412.02142v1), 2024-12-03

(36) [WSI-LLaVA: A Multimodal Large Language Model for Whole Slide Image](http://arxiv.org/abs/2412.02141v1), 2024-12-03

(37) [Trust & Safety of LLMs and LLMs in Trust & Safety](http://arxiv.org/abs/2412.02113v1), 2024-12-03

(38) [Explainable and Interpretable Multimodal Large Language Models: A Comprehensive Survey](http://arxiv.org/abs/2412.02104v1), 2024-12-03

(39) [Let's Think Var-by-Var: Large Language Models Enable Ad Hoc Probabilistic Reasoning](http://arxiv.org/abs/2412.02081v1), 2024-12-03

(40) [Leveraging Large Language Models to Democratize Access to Costly Financial Datasets for Academic Research](http://arxiv.org/abs/2412.02065v1), 2024-12-03

(41) [Future of Information Retrieval Research in the Age of Generative AI](http://arxiv.org/abs/2412.02043v1), 2024-12-03

(42) [LLMs4Life: Large Language Models for Ontology Learning in Life Sciences](http://arxiv.org/abs/2412.02035v1), 2024-12-02

(43) [PKRD-CoT: A Unified Chain-of-thought Prompting for Multi-Modal Large Language Models in Autonomous Driving](http://arxiv.org/abs/2412.02025v1), 2024-12-02

(44) [The use of large language models to enhance cancer clinical trial educational materials](http://arxiv.org/abs/2412.01955v1), 2024-12-02

(45) [The Reality of AI and Biorisk](http://arxiv.org/abs/2412.01946v1), 2024-12-02

(46) [A Shared Standard for Valid Measurement of Generative AI Systems' Capabilities, Risks, and Impacts](http://arxiv.org/abs/2412.01934v1), 2024-12-02

(47) [MALT: Improving Reasoning with Multi-Agent LLM Training](http://arxiv.org/abs/2412.01928v1), 2024-12-02

(48) [X-Prompt: Towards Universal In-Context Image Generation in Auto-Regressive Vision Language Foundation Models](http://arxiv.org/abs/2412.01824v1), 2024-12-02

(49) [[CLS] Attention is All You Need for Training-Free Visual Token Pruning: Make VLM Inference Faster](http://arxiv.org/abs/2412.01818v1), 2024-12-02

(50) [PhysGame: Uncovering Physical Commonsense Violations in Gameplay Videos](http://arxiv.org/abs/2412.01800v1), 2024-12-02

(51) [HackSynth: LLM Agent and Evaluation Framework for Autonomous Penetration Testing](http://arxiv.org/abs/2412.01778v1), 2024-12-02

(52) [Planning and Reasoning with 3D Deformable Objects for Hierarchical Text-to-3D Robotic Shaping](http://arxiv.org/abs/2412.01765v1), 2024-12-02

(53) [A Neurosymbolic Fast and Slow Architecture for Graph Coloring](http://arxiv.org/abs/2412.01752v1), 2024-12-02

(54) [Towards Resource Efficient and Interpretable Bias Mitigation in Large Language Models](http://arxiv.org/abs/2412.01711v1), 2024-12-02

(55) [Query Performance Explanation through Large Language Model for HTAP Systems](http://arxiv.org/abs/2412.01709v1), 2024-12-02

(56) [Are We There Yet? Revealing the Risks of Utilizing Large Language Models in Scholarly Peer Review](http://arxiv.org/abs/2412.01708v1), 2024-12-02

(57) [Unlocking Video-LLM via Agent-of-Thoughts Distillation](http://arxiv.org/abs/2412.01694v1), 2024-12-02

(58) [DaDu-E: Rethinking the Role of Large Language Model in Robotic Computing Pipeline](http://arxiv.org/abs/2412.01663v1), 2024-12-02

(59) [R-Bot: An LLM-based Query Rewrite System](http://arxiv.org/abs/2412.01661v1), 2024-12-02

(60) [PassionNet: An Innovative Framework for Duplicate and Conflicting Requirements Identification](http://arxiv.org/abs/2412.01657v1), 2024-12-02

(61) [Command-line Risk Classification using Transformer-based Neural Architectures](http://arxiv.org/abs/2412.01655v1), 2024-12-02

(62) [Using Large Language Models in Automatic Hint Ranking and Generation Tasks](http://arxiv.org/abs/2412.01626v1), 2024-12-02

(63) [NYT-Connections: A Deceptively Simple Text Classification Task that Stumps System-1 Thinkers](http://arxiv.org/abs/2412.01621v1), 2024-12-02

(64) [If Eleanor Rigby Had Met ChatGPT: A Study on Loneliness in a Post-LLM World](http://arxiv.org/abs/2412.01617v1), 2024-12-02

(65) [Medchain: Bridging the Gap Between LLM Agents and Clinical Practice through Interactive Sequential Benchmarking](http://arxiv.org/abs/2412.01605v1), 2024-12-02

(66) [Agentic-HLS: An agentic reasoning based high-level synthesis system using large language models (AI for EDA workshop 2024)](http://arxiv.org/abs/2412.01604v1), 2024-12-02

(67) [VideoLights: Feature Refinement and Cross-Task Alignment Transformer for Joint Video Highlight Detection and Moment Retrieval](http://arxiv.org/abs/2412.01558v1), 2024-12-02

(68) [SeqAfford: Sequential 3D Affordance Reasoning via Multimodal Large Language Model](http://arxiv.org/abs/2412.01550v1), 2024-12-02

(69) [Improved Large Language Model Jailbreak Detection via Pretrained Embeddings](http://arxiv.org/abs/2412.01547v1), 2024-12-02

(70) [Addressing Data Leakage in HumanEval Using Combinatorial Test Design](http://arxiv.org/abs/2412.01526v1), 2024-12-02

(71) [Data-Centric and Heterogeneity-Adaptive Sequence Parallelism for Efficient LLM Training](http://arxiv.org/abs/2412.01523v1), 2024-12-02

(72) [Scaling Law for Language Models Training Considering Batch Size](http://arxiv.org/abs/2412.01505v1), 2024-12-02

(73) [Intelligent Spark Agents: A Modular LangGraph Framework for Scalable, Visualized, and Enhanced Big Data Machine Learning Workflows](http://arxiv.org/abs/2412.01490v2), 2024-12-02

(74) [Early Exit Is a Natural Capability in Transformer-based Models: An Empirical Study on Early Exit without Joint Optimization](http://arxiv.org/abs/2412.01455v1), 2024-12-02

(75) [PLD+: Accelerating LLM inference by leveraging Language Model Artifacts](http://arxiv.org/abs/2412.01447v1), 2024-12-02

(76) [Multi-Facet Blending for Faceted Query-by-Example Retrieval](http://arxiv.org/abs/2412.01443v1), 2024-12-02

(77) [Learning Elementary Cellular Automata with Transformers](http://arxiv.org/abs/2412.01417v1), 2024-12-02

(78) [Efficient LLM Inference using Dynamic Input Pruning and Cache-Aware Masking](http://arxiv.org/abs/2412.01380v1), 2024-12-02

(79) [Adapting Large Language Models to Log Analysis with Interpretable Domain Knowledge](http://arxiv.org/abs/2412.01377v1), 2024-12-02

(80) [Understanding the World's Museums through Vision-Language Reasoning](http://arxiv.org/abs/2412.01370v1), 2024-12-02

(81) [Su-RoBERTa: A Semi-supervised Approach to Predicting Suicide Risk through Social Media using Base Language Models](http://arxiv.org/abs/2412.01353v1), 2024-12-02

(82) [MoTrans: Customized Motion Transfer with Text-driven Video Diffusion Models](http://dx.doi.org/10.1145/3664647.3680718), 2024-12-02

(83) [A 2-step Framework for Automated Literary Translation Evaluation: Its Promises and Pitfalls](http://arxiv.org/abs/2412.01340v1), 2024-12-02

(84) [Can Large Language Models Serve as Evaluators for Code Summarization?](http://arxiv.org/abs/2412.01333v1), 2024-12-02

(85) [The "LLM World of Words" English free association norms generated by large language models](http://arxiv.org/abs/2412.01330v1), 2024-12-02

(86) [The Seeds of the FUTURE Sprout from History: Fuzzing for Unveiling Vulnerabilities in Prospective Deep-Learning Libraries](http://arxiv.org/abs/2412.01317v1), 2024-12-02

(87) [RL2: Reinforce Large Language Model to Assist Safe Reinforcement Learning for Energy Management of Active Distribution Networks](http://arxiv.org/abs/2412.01303v1), 2024-12-02

(88) [LSceneLLM: Enhancing Large 3D Scene Understanding Using Adaptive Visual Preferences](http://arxiv.org/abs/2412.01292v1), 2024-12-02

(89) [Enhancing Perception Capabilities of Multimodal LLMs with Training-free Fusion](http://arxiv.org/abs/2412.01289v1), 2024-12-02

(90) [Align-KD: Distilling Cross-Modal Alignment Knowledge for Mobile Vision-Language Model](http://arxiv.org/abs/2412.01282v1), 2024-12-02

(91) [CPRM: A LLM-based Continual Pre-training Framework for Relevance Modeling in Commercial Search](http://arxiv.org/abs/2412.01269v2), 2024-12-02

(92) [Ponder & Press: Advancing Visual GUI Agent towards General Computer Control](http://arxiv.org/abs/2412.01268v1), 2024-12-02

(93) [Do Large Language Models with Reasoning and Acting Meet the Needs of Task-Oriented Dialogue?](http://arxiv.org/abs/2412.01262v1), 2024-12-02

(94) [Yi-Lightning Technical Report](http://arxiv.org/abs/2412.01253v2), 2024-12-02

(95) [Collaborative Instance Navigation: Leveraging Agent Self-Dialogue to Minimize User Input](http://arxiv.org/abs/2412.01250v1), 2024-12-02

(96) [Best Practices for Large Language Models in Radiology](http://arxiv.org/abs/2412.01233v1), 2024-12-02

(97) [GraphOTTER: Evolving LLM-based Graph Reasoning for Complex Table Question Answering](http://arxiv.org/abs/2412.01230v1), 2024-12-02

(98) [Composition of Experts: A Modular Compound AI System Leveraging Large Language Models](http://arxiv.org/abs/2412.01868v1), 2024-12-02

(99) [FD-LLM: Large Language Model for Fault Diagnosis of Machines](http://arxiv.org/abs/2412.01218v1), 2024-12-02

(100) [First numerical observation of the Berezinskii-Kosterlitz-Thouless transition in language models](http://arxiv.org/abs/2412.01212v1), 2024-12-02
